packages/web-components/src/components/va-pagination/test/va-pagination.e2e.ts

```diff
@@ -273,7 +273,7 @@ describe('uswds - va-pagination', () => {
                   <span class="usa-pagination__link-text">next</span>
                   <div id="next-arrow-icon">
                     <svg class="usa-icon" aria-hidden="true" role="img">
-                      <use href="/assets/sprite.svg#navigate_next"></use>
+                      <use href="/img/sprite.svg#navigate_next"></use>
```

Comment:
This doesn't seem to be related to the pill icon. Is there another issue that we should be linking to for reference?

Reply:
No, it's just a bug I found when simplifying the "where does the sprite sheet go" documentation. We had different copies of the sprite sheet in both the assets folder and img folder, and I was still updating the assets folder version even though it's no-longer being used, only to discover that the va-pagination component had never been updated to look for the img folder version (hope that makes sense?!?). So I updated the va-pagination component to look in the right place and got rid of the old duplicate so no one else makes the same mistake I did and update the wrong sprite sheet.